### DISCUSSION TITLE
What does elementLocked do?

### DISCUSSION THREAD

**Discussion by mwufi** [excalidraw/excalidraw#2523] [Q&A] [answered]
I can't quite seem to get it to do anything xD. I mean, I can toggle the lock icon, but nothing really happens?

**Comment by dwelle** [comment, marked as answer]
It locks the tool(s) so that it doesn't reset to selection after you create an element. It's not very intuitive, I agree :(.

**Reply by crashedpixel** [reply]
@dwelle is there a work around you can think of where you could lock all elements from being selected so when the user clicks it just pans instead of selecting an element?

**Reply by dwelle** [reply]
Depends on what you're after.

- if presenting/viewing, you can use view mode.

	Right-click on canvas and choose view mode:

	![image](https://user-images.githubusercontent.com/5153846/147277441-ab3e6ece-10e0-4be2-ac32-f19b9bf694be.png)

- if to prevent accidental modification when you need to move around canvas, just drag by holding mousewheel, or space-drag. We will also add `pan`/`hand` tool at some point.
- To prevent specific elements to be edited (e.g. background images/containers), we'll support element locking soon (https://github.com/excalidraw/excalidraw/issues/1976).
